build.fsx

```diff
@@ -219,7 +221,43 @@ Target.create "Restore" (fun _ -> DotNet.restore dotnetSimple "OpenTK.sln" |> ig
 Target.create "AssemblyInfo" (fun _ ->
     //TODO: Create and update the Directory.Build.props file with the version information taken from the releaseNotes value.
     // see https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2019#directorybuildprops-and-directorybuildtargets
-    Trace.traceError "Unimplemented.")
+
+    let getAssemblyInfoAttributes (projectName:string) =
+        [ AssemblyInfo.Title (projectName)
+          AssemblyInfo.Product project
+          AssemblyInfo.Description summary
+          AssemblyInfo.Version release.AssemblyVersion
+          AssemblyInfo.FileVersion release.AssemblyVersion
+          AssemblyInfo.CLSCompliant true
+          AssemblyInfo.Copyright copyright
+        ]
+
+    let getProjectDetails (projectPath:string) =
+        let projectName = System.IO.Path.GetFileNameWithoutExtension(projectPath)
+        ( projectPath,
+          projectName,
+          System.IO.Path.GetDirectoryName(projectPath),
+          (getAssemblyInfoAttributes projectName)
+        )
```

Comment:
This is a good candidate for an anonymous record instead:
Try replace this with:
```fsharp
{|
    ProjectPath = projectPath
    ProjectName = projectName
    DirectoryName = System.IO.Path.GetDirectoryName(projectPath)
    AssemblyInfoAttributes = getAssemblyInfoAttributes projectName
|}
```

Then you can access the record fields by name later on for a bit more simplicity/clarity.